nym-api/src/circulating_supply_api/handlers.rs

```diff
@@ -12,21 +11,25 @@ use nym_validator_client::nyxd::Coin;
 
 pub(crate) fn circulating_supply_routes() -> Router<AxumAppState> {
     Router::new()
+        .route("/", axum::routing::get(get_full_circulating_supply))
         .route(
-            &static_routes::v1::circulating_supply(),
-            axum::routing::get(get_full_circulating_supply),
-        )
-        .route(
-            &static_routes::v1::circulating_supply::circulating_supply_value(),
+            "/total-supply-value",
```

Comment:
just curious, why did you opt to go back to hardcoded routes rather than using the values you defined previously?

Reply:
(if it works, it works, no need to change it now)

Reply:
> why did you opt to go back to hardcoded routes rather than using the values you defined previously?

because of paths with parameters. They are defined in axum as 
- `/v1/api/path/:param`

while in utoipa annotations paths look like this
- `/v1/api/path/{param}`

which means I cannot reuse the predefined constants. If one is predefined as a constant, and I have to rewrite the other, then I don't see a point of having a constant at all.